### PR TITLE
[RFC] 0017 Deprecation of log.original Stage 3 Proposal

### PR DESCRIPTION
**Pull request by djptek**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? Y 
- Have you followed the [contributor guidelines](https://github.com/elastic/ecs/blob/master/CONTRIBUTING.md)? Y 
- For proposing substantial changes or additions to the schema, have you reviewed the [RFC process](https://github.com/elastic/ecs/blob/master/rfcs/README.md)? Y 
- If submitting code/script changes, have you verified all tests pass locally using `make test`? n/a
- If submitting schema/fields updates, have you generated new artifacts by running `make` and committed those changes? n/a
- Is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed. Y
- Have you added an entry to the [CHANGELOG.next.md](https://github.com/elastic/ecs/blob/master/CHANGELOG.next.md)? TBC

### CODE REVIEW

File: rfcs/text/0017-remove-log-original.md [hunk 1]
```diff
@@ -58,9 +64,13 @@ The removal of `log.original` will be considered a breaking change since the fie
...
 
-- The logs UI `message` column currently displays `log.original` in the absence of a `message` field. This should be updated to use `event.original` as the substitute field. See [builtin_rules](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.ts) and [associated test](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.test.ts).
+- The logs UI `message` column currently displays `log.original` in the absence of a `message` field. It should be updated to use `event.original` as the substitute field. See [builtin_rules](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.ts) and [associated test](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.test.ts). Kibana team is aware and has a [PR awaiting review](https://github.com/elastic/kibana/pull/102236) which addresses this.
+
+- The breakdown of usage of this field in Kibana source shows:
```
Comment (kgeller) [inline rfcs/text/0017-remove-log-original.md:69]: Is it possible to fix the indentation here? 
Screen Shot 2021-06-21 at 8 51 45 AM

Comment (djptek) [reply rfcs/text/0017-remove-log-original.md:69]: Thanks @kgeller, fixed : 
image

**Review by kgeller — APPROVED** [review]
LGTM

File: rfcs/text/0017-remove-log-original.md [hunk 2]
```diff
@@ -89,8 +95,12 @@ The following are the people that consulted on the contents of this RFC.
...
-* [#841](https://github.com/elastic/ecs/issues/841)
-* [#777](https://github.com/elastic/integrations/issues/777)
+### Issues
+* [ecs #841](https://github.com/elastic/ecs/issues/841)
+* [integrations #777](https://github.com/elastic/integrations/issues/777)
+
+### Stack PRs
+* [kibana PR #102236](https://github.com/elastic/kibana/pull/102236)
 
 ### RFC Pull Requests
```
Comment (ebeahan) [inline rfcs/text/0017-remove-log-original.md:105]: Can you add the stage 3 PR here as well?

File: rfcs/text/0017-remove-log-original.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # 0017: Remove log.original
 
-- Stage: **2 (candidate)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
-- Date: **2021-04-28** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+- Stage: **3 (candidate)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```
Comment (ebeahan) [inline rfcs/text/0017-remove-log-original.md:3]:
```suggestion
- Stage: **3 (finished)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

File: rfcs/text/0017-remove-log-original.md [hunk 4]
```diff
@@ -72,10 +82,6 @@ The removal of `log.original` will be considered a breaking change since the fie
 
 As a breaking change, this would require timely communication to the Elastic Community.
```
Comment (ebeahan) [inline rfcs/text/0017-remove-log-original.md:83]: Let's note the resolution taken to address this concern.

Can you describe the work you've done to communicate this change to internal stakeholders, @djptek?

**Comment by ebeahan** [comment]
A thought I had while reviewing: 

We will want to update the `log.original` field description indicating the field's deprecation and upcoming removal from ECS.

File: rfcs/text/0017-remove-log-original.md (hunk 4 above)
Comment (djptek) [reply rfcs/text/0017-remove-log-original.md:80]: added detail

File: rfcs/text/0017-remove-log-original.md (hunk 2 above)
Comment (djptek) [reply rfcs/text/0017-remove-log-original.md:111]: Added stage 3 PR

**Comment by djptek** [comment]
>> We will want to update the log.original field description indicating the field's deprecation and upcoming removal from ECS.

This is referenced at L17 in the RFC and proposed change is here: https://github.com/djptek/ecs/blob/rfc_0017_stage_3/rfcs/text/0017/event.yml

**Comment by djptek** [comment]
@elasticmachine run elasticsearch-ci/docs

File: rfcs/text/0017-remove-log-original.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # 0017: Remove log.original
 
-- Stage: **2 (candidate)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
-- Date: **2021-04-28** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+- Stage: **3 (finished)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **2021-06-18** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
```
Comment (ebeahan) [inline rfcs/text/0017-remove-log-original.md:4]: Update when you are ready to merge 🚢

**Review by ebeahan — APPROVED** [review]
LGTM 👍

File: rfcs/text/0017-remove-log-original.md (hunk 5 above)
Comment (djptek) [reply rfcs/text/0017-remove-log-original.md]: thx! done, just waiting for last run thru ci...

**Pull request merged**